resources/zib/zib-NursingIntervention.xml

```diff
@@ -0,0 +1,437 @@
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.12" />
+        <comment value="ProcedureEndDateTime" />
+      </mapping>
+    </element>
+    <element id="CarePlan.contributor">
+      <path value="CarePlan.contributor" />
+      <slicing>
+        <discriminator>
+          <type value="exists" />
+          <path value="extension(url='http://nictiz.nl/fhir/StructureDefinition/ext-NursingIntervention-ContributorIsRequester')" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CarePlan.contributor:requester">
+      <path value="CarePlan.contributor" />
+      <sliceName value="requester" />
+      <short value="Requester" />
+      <definition value="The health professional who requested the nursing intervention. If desired, only the requester’s specialty can be entered." />
+      <comment value="Each occurence of the zib HealthProfessional is normally represented by _two_ FHIR resources: a PractitionerRole resource (instance of [zib-HealthProfessional-PractitionerRole](http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-PractitionerRole)) and a Practitioner resource (instance of [zib-HealthProfessional-Practitioner](http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-Practitioner)). The reference to the Practitioner resource is referenced from the PractitionerRole instance. For this reason, sending systems should fill the reference to the PractitionerRole instance here, and not the Practitioner resource.&#xD;&#xA;&#xD;&#xA;In rare circumstances, there is only a Practitioner instance, in which case it is that instance which will be referenced here." />
+      <alias value="Aanvrager" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthcareProvider-PractitionerRole" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthcareProvider-Practitioner" />
+      </type>
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.8" />
+        <comment value="Requester" />
+      </mapping>
+    </element>
+    <element id="CarePlan.contributor:requester.extension">
+      <path value="CarePlan.contributor.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CarePlan.contributor:requester.extension:contributorIsRequester">
+      <path value="CarePlan.contributor.extension" />
+      <sliceName value="contributorIsRequester" />
+      <min value="1" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-NursingIntervention-ContributorIsRequester" />

```

Comment:
See comment on extension on `id`

Reply:
This extension doesn't represent a specific concept from the zib.

Reply:
See other comment